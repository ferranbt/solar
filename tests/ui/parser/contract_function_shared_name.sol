contract C {
    function C() public {} //~ ERROR: functions are not allowed to have the same name as the contract

    function c() public {}
}
