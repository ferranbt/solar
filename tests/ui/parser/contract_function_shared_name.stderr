error: functions are not allowed to have the same name as the contract
  --> ROOT/tests/ui/parser/contract_function_shared_name.sol:LL:CC
   |
LL |     function C() public {}
   |              ^
   |
   = note: if you intend this to be a constructor, use "constructor(...) { ... }" to define it

error: aborting due to 1 previous error

